### PR TITLE
Docs: Usage script readability

### PR DESCRIPTION
**Pull request by Meldiron**
This PR makes usage example scripts:

- More readable for humans
- Copyable from readme
- Copyable from terminal

### CODE REVIEW

**Comment by Meldiron** [comment]
Tested all scripts manually

File: runtimes/dotnet-3.1/README.md [hunk 1]
```diff
@@ -12,7 +12,17 @@ To learn more about runtimes, visit [Structure](https://github.com/open-runtimes
...
 ```bash
 mkdir dotnet-or && cd dotnet-or
-printf "using System.Threading.Tasks;\nusing System.Collections.Generic;\n\npublic async Task<RuntimeResponse> Main(RuntimeRequest req, RuntimeResponse res) => res.Json(new Dictionary<string, object>() {{ \"n\", new System.Random().NextDouble() }} );" > Index.cs
+tee -a Index.cs << END
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+public async Task<RuntimeResponse> Main(RuntimeRequest req, RuntimeResponse res)
+{
+    return res.Json(new Dictionary<string, object>() {{ "n", new System.Random().NextDouble() }} );
```
Comment (gewenyu99) [inline runtimes/dotnet-3.1/README.md:21]: We can probably format this better, now that we use `tee`

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
I'm good with this, 1 comment. Only concern is that you clearly explain what `tee` does in the instructions preceding this code block.

**Comment by Meldiron** [comment]
@gewenyu99 above script we say `Add code into "Index.cs" file`. Do you think that is good enough or we should be more specific?

**Comment by gewenyu99** [comment]
> @gewenyu99 above script we say `Add code into "Index.cs" file`. Do you think that is good enough or we should be more specific?

Good enough for now :)

**Review by gewenyu99 — APPROVED** [review]
LGTM

**Comment by Meldiron** [comment]
This will be broken by V3. New branch should be started when v3 is merged.

**Comment by Meldiron** [comment]
Closing in favour of: https://github.com/open-runtimes/open-runtimes/pull/154

**Pull request closed without merging**